config/application.yml.default

```diff
@@ -126,6 +126,8 @@ idv_send_link_attempt_window_in_minutes: 10
 idv_send_link_max_attempts: 5
 ie11_support_end_date: '2022-12-31'
 idv_sp_required: false
+in_person_cta_variant_testing_enabled: true
+in_person_cta_variant_testing_percents: '{"A":50, "B":50}'
```

Comment:
Are these the values we want to have live in production when we deploy? Or will we be setting them manually?

Reply:
Yes. Per Sumi, we want a 50/50 split until Feb 1. Starting Feb 1 we want A:12.5,B:12.5,C:75 as in [the throttling PR](https://github.com/18F/identity-idp/pull/7693).